includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -673,6 +646,52 @@ public function stats_boxes() {
 		return apply_filters( 'sensei_analysis_stats_boxes', $stats_to_render );
 	}
 
+	/**
+	 * Get the last user activity date.
+	 * It is based on the date on which the last lesson was submitted (completed).
+	 *
+	 * @since 4.2.0
+	 *
+	 * @param  int $user_id
+	 * @return string
```

Comment:
🤦 I will stop missing these one day, I promise! Fixed in d0575dbd38c46d5cd3d01d9ee29e22bdca1756ff.

Reply:
Just noticed that the "param comments check" is disabled in [phpcs](https://github.com/Automattic/sensei/blob/master/phpcs.xml.dist#L69). Maybe we should enable it? I'm not sure if there is a rule for `@return`.

Reply:
> Just noticed that the "param comments check" is disabled in [phpcs](https://github.com/Automattic/sensei/blob/master/phpcs.xml.dist#L69). Maybe we should enable it?

Makes sense, so long as it would only impact changed code and not check the entire codebase.

Reply:
👍

 PR: https://github.com/Automattic/sensei/pull/4877